lib/elixometer.ex

```diff
@@ -122,11 +122,12 @@ defmodule Elixometer do
     end
   end
 
-  defp build_timer_body(%Timer{} = timer_data) do
-    quote do
-      timed(unquote(timer_data.key), unquote(timer_data.units)) do
-        unquote(timer_data.body)
-      end
+  defp build_timer_body(%Timer{body: [do: block]} = timer) do
+    build_timer_body(%{timer | body: block})
```

Comment:
I adopted this approach, but it also makes me wonder if something like this would also work:

```elixir
defp normalize_body(nil), do: raise "timed function must have a body"
defp normalize_body(body) when is_list(body), do: body
defp normalize_body(body), do: [do: body]
```

Are there cases where the `is_list(body)` check is ambiguous? The goal here is to remove the version check.